system/assets/js/admin/specific/datamanager/quickAddForm/quickAddFormHandler.js

```diff
@@ -72,11 +72,17 @@
 
 	resetForm = function(){
 		$quickAddForm.trigger( "reset" );
-		for (instance in CKEDITOR.instances){
-			CKEDITOR.instances[instance].setData("");
-		}
+		clearRicheditor();
 	};
 
+	clearRicheditor = function() {
```

Comment:
`clearRicheditor` is not `var` scoped. It should be added to the list of `var` scoped variables at the top of the function (line 4).